### PR TITLE
`0.8.10`

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by michaelweinold** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,16 +20,16 @@ requirements:
     - python >=3.6
     - setuptools
   run:
-    - bw_migrations
+    - python >=3.6
     - bw2calc >=1.7.4
     - bw2data >=3.5.1
+    - bw_migrations
```
Comment (michaelweinold) [inline recipe/meta.yaml:26]: ...moved these packages to resemble the list in the [`setup.py` of `bw2io`](https://github.com/brightway-lca/brightway2-io/blob/0.8.10/setup.py):

**Comment by m-rossi** [comment]
I am confused, what was the intention of this PR?

**Comment by michaelweinold** [comment]
Yes, sorry - had not synced my own fork. Only the `brightway2` feedstock needs updating, this one is fine.

**Pull request closed without merging**